homeassistant/components/switch/tplink.py

```diff
@@ -34,17 +36,19 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     from pyHS100 import SmartPlug
     host = config.get(CONF_HOST)
     name = config.get(CONF_NAME)
+    leds_on = config.get(CONF_LEDS, True)
```

Comment:
`CONF_LEDS` needs to be added to the configuration validation and the default set there.